server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -153,30 +166,6 @@ public class SecretResourceTest {
   //---------------------------------------------------------------------------------------
   // createOrUpdateSecret
   //---------------------------------------------------------------------------------------
-
-  @Test public void createOrUpdateSecretPreservesExistingNullSecretOwner() throws Exception {
-    String secretName = UUID.randomUUID().toString();
-
-    CreateSecretRequestV2 createRequest = CreateSecretRequestV2.builder()
-        .name(secretName)
-        .content(encode("foo"))
-        .build();
-    create(createRequest);
-
-    SecretDetailResponseV2 originalDetails = lookup(secretName);
-    assertNull(originalDetails.owner());
-
-    CreateOrUpdateSecretRequestV2 updateRequest = CreateOrUpdateSecretRequestV2.builder()
-        .content(encode("bar"))
-        .description(UUID.randomUUID().toString())
-        .build();
-
-    assertEquals(201, createOrUpdate(updateRequest, secretName).code());
-
-    SecretDetailResponseV2 updatedDetails = lookup(secretName);
-    assertNull(updatedDetails.owner());
-  }
-
```

Comment:
the test config file now specifies that the owner should be inferred so this test does not make sense any more

Reply:
I will mark it as @Ignored actually